<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Amilaths-MacBook-Pro.local" failures="3" tests="3" name="fashionette Test cases" time="94.692" errors="0" timestamp="28 Nov 2021 22:45:52 GMT">
  <testcase classname="com.fashionette.tests.AddToCartTest" name="verifyProductAddedToCart" time="28.315">
    <failure type="java.lang.AssertionError" message="Item was not added to cart expected [BEREITS IM WARENKORB] but found [IN DEN WARENKORB]">
      <![CDATA[java.lang.AssertionError: Item was not added to cart expected [BEREITS IM WARENKORB] but found [IN DEN WARENKORB]
	at com.fashionette.tests.AddToCartTest.verifyProductAddedToCart(AddToCartTest.java:56)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- verifyProductAddedToCart -->
  <testcase classname="com.fashionette.tests.ApplyVoucherTest" name="verifyVoucherApplied" time="23.65">
    <failure type="java.lang.AssertionError" message="Coupon code did not applied expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: Coupon code did not applied expected [true] but found [false]
	at com.fashionette.tests.ApplyVoucherTest.verifyVoucherApplied(ApplyVoucherTest.java:41)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verifyVoucherApplied -->
  <testcase classname="com.fashionette.tests.ModifyUserDataTest" name="verifyUserDataGetModified" time="13.131">
    <failure type="java.lang.AssertionError" message="user data is not modified  expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: user data is not modified  expected [true] but found [false]
	at com.fashionette.tests.ModifyUserDataTest.verifyUserDataGetModified(ModifyUserDataTest.java:40)
	at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
	at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
	at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
	at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
	at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
	at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
	at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
... Removed 24 stack frames]]>
    </failure>
  </testcase> <!-- verifyUserDataGetModified -->
</testsuite> <!-- fashionette Test cases -->
